l10n_it_financial_statements_report/report/financial_statements_report_xlsx.py

```diff
@@ -566,7 +566,9 @@ def get_write_data(self, line, col_dict, report, report_data, report_result):
             allow = True
 
         if value:
-            if isinstance(value, int | float) and cell_type not in (
+            if (
+                isinstance(value, int) or isinstance(value, float)
```

Comment:
Giusto per info, si potrebbe anche usare `isinstance(value, (int, float))`:
> **isinstance(object, classinfo)**
[...]
If classinfo is a tuple of type objects (or recursively, other such tuples), return True if object is an instance of any of the types.

(da https://docs.python.org/3.7/library/functions.html#isinstance)
ma non vedo problemi con l'implementazione attuale